math_functions/lib/math_functions.ex

```diff
@@ -0,0 +1,24 @@
+defmodule MathFunctions do
+  
```

Comment:
If you wanted to be fancy, to could factor these and same a lot of multiplying:

(untested, so ...)

~~~ elixir
  def sin(x) do
    x2 = x*x
    x*(1 - x2/6*(1-x2/20*(1-x2/42*(1-x2/72))))
  end
~~~